### ISSUE TITLE
August team coordination

### ISSUE THREAD

**Issue by skmp** [reicast/reicast-emulator#1242]
Location
--
- #reicast-dev in http://chat.reicast.com

Media
--
- Primary text.
- I'm open to experimenting with the voice/video functionality in discord vs just text-based IRC style. Let's keep in mind that, people are often uncomfortable to use voice or have network issues though.

Agenda
---

We've had a lot of new volunteers, and a lot of new ideas over the past 2-3 weeks. 

Following the discussions on the chat and issues, it looks like everyone's moving on a random direction at this point, and it's a bit hard to agree on anything.

I think it'd make sense for all of us to be around discord on Friday, 2nd of august, and discuss what everyone wants to do, and figure out some common goals we can coordinate on.

From my todo list
- [x] Coordinate R7 release. Should happen this weekend. See #571 
- [ ] Getting started guide for contribs (#1102), further project documentation
- [ ] We need a designer. I have a reference, I will follow up and report by friday.
(I figured, since no one else is willing to propose a day/time, I'll just go ahead and propose one?)
- [ ] Code formatting (#1240)
- [ ] FPGA update (is the board out yet?)

I need a place to put my todos
- [ ] Follow up #1225 
- [ ] Collaboration / Merging to master etc. / How to work effectively within the team 
- [ ] Code of Conduct
- [ ] Legal entity update

@reicast/members @flyinghead @twinaphex @rrrh @baka0815 @pold500 @davidgfnet

**Comment by baka0815** [comment]
Yes, I am (just created an account, haven't used before): slack-at-baka0815.de

**Comment by pold500** [comment]
I'm available via discord.

**Comment by skmp** [comment]
@baka0815 err sorry, my bad, I meant discord*. I keep forgetting we have moved out from slack. (See http://chat.reicast.com)

**Comment by skmp** [comment]
So, here are a few updates
- Released r7 (see https://github.com/reicast/reicast-emulator/releases/tag/r7)
- Decided next releases will be on Aug 20th, Sept 8th. Let's make sure `master` is in a great shape in these days.
- Decided will auto-tag beta at 25th of each month, and auto-tag release at 8th of each month.

**Comment by baka0815** [comment]
Looks like I kann kill my slack account. :D
Don't have a Discord account yet, will create one in the upcoming days.

**Comment by spielvan** [comment]
@skmp , reicast on android has improved a lot, the advance that has had since the creation of the beta tester system is incredible, I believe that when I can do the graph for 1x, 2x, 3x .... will be definitive and solve the problems of bugs graphics, I have impression that they occur due to a very high demand on the GPU. A hug and success always.

**Comment by spielvan** [comment]
skmp Congratulations on the reicast port for the retroarch android 32bits, I already got the core for my retroarch 1.7.3 "Reicast 0.1 689e21a", so that gave a problem, when I load a game, which usually squeezes into the reicast separate version of the retroarch, it loads the dreamcast bios normally, but when it goes to load the game, it closes the retroarch and goes to the android tv box.

**Comment by twinaphex** [comment]
Please don't emburden core upstream authors with this, they don't like being emburdened with these kind of questions. You should direct them at us instead.

**Comment by spielvan** [comment]
@twinaphex  I'm sorry, I was so excited about the news that I had not even noticed this detail, I'll post it on the retroarch forum.

**Comment by twinaphex** [comment]
@spielvan No problem.
